BoxContentSDK/BoxContentSDK/Requests/BOXSearchRequest.m

```diff
@@ -159,6 +159,20 @@ - (void)performRequestWithCompletion:(BOXItemArrayCompletionBlock)completionBloc
     }
 }
 
+- (void)performRequestWithCached:(BOXItemArrayCompletionBlock)cacheBlock
+                       refreshed:(BOXItemArrayCompletionBlock)refreshBlock
+{
+    if (cacheBlock) {
+        if ([self.cacheClient respondsToSelector:@selector(retrieveCacheForSearchRequest:completionBlock:)]) {
+            [self.cacheClient retrieveCacheForSearchRequest:self completionBlock:cacheBlock];
+        } else {
+            cacheBlock(nil, 0, NSMakeRange(0, 0), nil);
+        }
+    }
+
+    [self performRequestWithCompletion:refreshBlock];
```

Comment:
Do we need to be hooking up the cacheClient to performRequestWithCompletion: so that developers have an opportunity to write to their cache, when the server responds with results?